Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -9,10 +9,16 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 ExitOnPRBuild
 
 . $PSScriptRoot/SonarQubePostTestImpl.ps1
-. $PSScriptRoot/SonarQubeBuildBreaker.ps1
 . $PSScriptRoot/PRCA/Orchestrator.ps1
 
+. $PSScriptRoot/SonarQubeMetrics.ps1
+. $PSScriptRoot/SonarQubeReportHandler.ps1
+. $PSScriptRoot/SonarQubeBuildBreaker.ps1
+
+
 InvokeMSBuildRunnerPostTest
-UploadSummaryMdReport
+# PRCA
 HandleCodeAnalysisReporting
+
+UploadSummaryMdReport
```

Comment:
Either ordering seems equally valid so there's no particular reason to change it back.
The more important point is that you thought it was worth making the change but left reviewers to guess/ask what the reason was.